classes/pdf/PDFGenerator.php

```diff
@@ -147,7 +148,7 @@ public function setFontForLang($iso_lang)
         if (array_key_exists($iso_lang, $this->font_by_lang)) {
             $this->font = $this->font_by_lang[$iso_lang];
         }else {
-            $this->font = self::DEFAULT_FONT;
+            $this->font = 'dejavusans';
```

Comment:
@cdoublev this line will fix the problem, cause, if the lang chosen does not exist in the $font_by_lang array, then it will assign the 'dejavusans' as font and not the DEFAULT_FONT